index.js

```diff
@@ -52,7 +52,88 @@ server.get("/artigo/:slug", (request, response) => {
 });
 
 server.get("/login", (request, response) => {
-  response.sendFile(__dirname+"/views/login.html");
+  const html = jetpack.read(__dirname+"/views/login.html");
+  const feedback = "";
+  const formattedHtml = Eta.render( html, feedback );
+  response.send(formattedHtml);
+});
+
+server.post("/login", (request, response) => {
+  const html = jetpack.read(__dirname+"/views/login.html");
+  const database = JSON.parse(jetpack.read('./database.json'));
+  const userLog = request.cookies.user;
+  const dataReq = request.body;
+  const comparative = database.users[dataReq.email];
+
+  if(comparative) {
```

Comment:
esse nome ficou confuso, poderia ser

```suggestion
  const isEmailExistent = database.users[dataReq.email];

  if(isEmailExistent) {
```